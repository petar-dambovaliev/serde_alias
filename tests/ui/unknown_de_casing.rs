extern crate serde_alias;

use serde::{Deserialize, Serialize};
use serde_alias::serde_alias;

#[serde_alias(camelCase)]
#[derive(Serialize, Deserialize, Debug, Eq, PartialEq)]
struct Foo {
    bar: String,
    bar1: String,
    bar2: String,
    bar3: String,
    bar4: String,
}

fn main() {}
