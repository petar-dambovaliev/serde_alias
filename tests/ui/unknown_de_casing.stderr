error: custom attribute panicked
 --> $DIR/unknown_de_casing.rs:6:1
  |
6 | #[serde_alias(camelCase)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: unknown casing: camelCase try one of ["PascalCase", "CamelCase", "LowerCase", "UpperCase", "SnakeCase", "ScreamingSnakeCase", "KebabCase", "ScreamingKebabCase"]
